src/styles/_card.scss

```diff
@@ -0,0 +1,124 @@
+.container {
+  display: grid;
+  justify-content: center;
+
+  $grid-columns: 1;
+
+  grid-template-columns: repeat($grid-columns, 200px);
+  padding: 50px 40px;
+  gap: 46px 48px;
+
+  @media (min-width: 488px) {
+    $grid-columns: 2;
+
+    grid-template-columns: repeat($grid-columns, 200px);
+  }
+
+  @media (min-width: 768px) {
+    $grid-columns: 3;
+
+    grid-template-columns: repeat($grid-columns, 200px);
+  }
+
+  @media (min-width: 1024px) {
+    $grid-columns: 4;
+
+    grid-template-columns: repeat($grid-columns, 200px);
+  }
+}
+
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  border: 1px solid $border-color;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), hex color codes should be replaced with variables if they are used multiple times or represent a theme color. Ensure `$border-color` is defined in `_variables.scss`.